front_end/content.py

```diff
@@ -363,14 +447,14 @@ def get_problem_ids(self, course_id, assignment_id):
     def get_courses(self, show_hidden=True):
```

Comment:
In this case, I would prefer to make a separate function called get_courses_details. get_courses() is used on every page in the navbar, so we want it to be simple and fast. However, in this case, we are pulling details about the course, and this is only used occasionally. For now get_courses_details can just return course_id, title, visible, and introduction. But in the future, we may want it to pull additional details.

Reply:
Did I just start using it to assign "registered_courses" for admins? I'm confused on this one.